tools/triage.sh

```diff
@@ -0,0 +1,205 @@
+    
+    new_all_approvers="$(echo -e "$all_approvers\n$approvers" | sort -u)"
+    if [ "$all_approvers" != "$new_all_approvers" ]
+    then
+        # this isn't technically a problem if they all have an approver in common, ie super_approvers is non-empty, suppress warning?
+        if [ "$all_approvers" != "" ]
+        then
+            echo "manual: multiple XEPs modified with different approvers, split?"
+            super_approvers="$(comm -12 <(echo "$super_approvers") <(echo "$approvers") | sort -u)"
+        else
+            super_approvers="$new_all_approvers"
+        fi
+        all_approvers="$new_all_approvers"
+    fi
+    
+    # check revision block added and minor version bump
+    versions="$(xpath "$xep" '/xep/header/revision/version/text()')"
+    latest_num_versions="$(echo "$versions" | wc -l)"
+    latest_version="$(echo "$versions" | head -n1)"
+    # this git-cat-file usage assumes no entities etc are ever removed from xep.ent etc, this should be true
+    # but if it ever isn't, then we'll have to check out the entire repo as of $merge_base to do that
+    versions="$(git cat-file -p "$merge_base:./$xep" | xpath - '/xep/header/revision/version/text()')"
+    merge_base_num_versions="$(echo "$versions" | wc -l)"
+    merge_base_version="$(echo "$versions" | head -n1)"
+    if [ $latest_num_versions -gt $merge_base_num_versions ]
+    then
+        if ! echo "$latest_version" | grep -E '^[0-9]+\.[0-9]+\.[0-9]+$' &>/dev/null
+        then
+            echo "error: version is not semver format: '$latest_version'"
+            exit 1
+        fi
+        if [ "$latest_version" == "$merge_base_version" ]
+        then
+            echo "tag: needs-version-block"
+            # todo: anyone have a fancy way to check semver is higher in $latest_version ?
+            # beware we have funky non-semver versions
+            # for xep in xep-*.xml; do xpath "$xep" '/xep/header/revision/version/text()'; done | sort -u > xep_versions.txt
+        fi
+    else
+        echo "tag: needs-version-block"
+    fi
+done
+
+if [ "$super_approvers" != "" ]
+then
+    pr_authors="$(git --no-pager log --format="%aE" "$merge_base".."$pr_commit" | sort -u)"
+    pr_authors_super_approvers="$(comm -12 <(echo "$super_approvers") <(echo "$pr_authors") | sort -u)"
+    if [ "$pr_authors_super_approvers" != "" ]
+    then
+        echo "info: entire PR can be approved by one of the commit authors: '$(echo "$pr_authors" | tr '\n' ' ' | xargs)'"
+    fi
+    echo "info: entire PR can be approved by: '$(echo "$super_approvers" | tr '\n' ' ' | xargs)'"
+fi
+
+if echo "$all_approvers" | grep '^Council$' &>/dev/null
+then
+    echo "tag: needs-council"
+fi
+
+if echo "$all_approvers" | grep '^Board$' &>/dev/null
+then
+    echo "tag: needs-board"
+fi
+
+if echo "$all_approvers" | grep '^Unknown$' &>/dev/null
+then
+    echo "tag: needs-unknown-approver"
+    echo "warning: approver is unknown, perhaps old XEP that needs fixed?"
```

Comment:
This should be error/manual, shouldn't it?

Reply:
I'd prefer to just put an `<approver/>` in all old XEPs and remove this

Reply:
205 XEPs are missing approver :skull:

Reply:
In favour of "error" then.